ntbs-service/DataMigration/NotificationImportRepository.cs

```diff
@@ -21,6 +23,11 @@ public NotificationImportRepository(NtbsContext context)
             _context = context;
         }
 
+        public void AddSystemUserToAudits()
+        {
+            _context.AddAuditCustomField(CustomFields.AppUser, AuditService.AuditUserSystem);

```

Comment:
This is ok (especially since there is precedent for doing this)... however, it would be nice if this logic was handled during the injection of the context (i.e. we either do this or we don't depending on whether we're in a hangfire job or not). That would mean that we could isolate it to one place and then never have to worry about it again. That's definitely tech debt though.